cmd/ronin/accountcmd.go

```diff
@@ -357,3 +374,27 @@ func accountImport(ctx *cli.Context) error {
 	fmt.Printf("Address: {%x}\n", acct.Address)
 	return nil
 }
+
+func accountCheck(ctx *cli.Context) error {
+	keyfile := ctx.Args().First()
+	if len(keyfile) == 0 {
+		utils.Fatalf("keyfile must be given as argument")
+	}
+	key, err := crypto.LoadECDSA(keyfile)
+	if err != nil {
+		utils.Fatalf("Failed to load the private key: %v", err)
+	}
+	address := crypto.PubkeyToAddress(key.PublicKey)
+
+	stack, _ := makeConfigNode(ctx)
+	for _, wallet := range stack.AccountManager().Wallets() {
+		for _, account := range wallet.Accounts() {
+			if account.Address == address {
+				fmt.Printf("Found account %x\n", address)
```

Comment:
Should we rename this message for more clear like

Reply:
This subcommand is used to check if the account exists, when using the subcommand in bash, we will print a clearer message

Reply:
Oke I see the message in below logic